File: 412. Fizz Buzz/if.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> res;
        int val;
        for (int i = 0; i < n; i++) {
            val = i + 1;
            if ((val % 3 == 0) && (val % 5 == 0))
                res.push_back("FizzBuzz");
            else if (val % 3 == 0)
                res.push_back("Fizz");
            else if (val % 5 == 0)
                res.push_back("Buzz");
            else
                res.push_back(to_string(val));
        }

        return res;
    }
};

File: 412. Fizz Buzz/if.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        res = []

        for i in range(n):
            val = i + 1

            if ((val % 3 == 0) and (val % 5 == 0)):
                res.append("FizzBuzz")
            elif val % 3 == 0:
                res.append("Fizz")
            elif val % 5 == 0:
                res.append("Buzz")
            else:
                res.append(str(val))

        return res
